packages/management-api/src/managed-pod.ts

```diff
@@ -108,6 +121,36 @@ export class ManagedPod {
     return this._management
   }
 
+  getManagementError(): Error | null {
+    if (!this._management.status.error) return null
+
+    return new Error(`(${this._management.status.error.code}): ${this._management.status.error.message}`)
```

Comment:
Is this format intended? `(${this._management.status.error.code})` should be like: `(404):Not found` (the parentheses)

Reply:
Yes. I though it made use of the http code and seems to be consistent with the way HTTP Errors are displayed when converted to strings. Happy to change if preferred.

Reply:
OK, that's fine if it's intentional.